googletest/include/gtest/internal/gtest-port.h

```diff
@@ -938,11 +940,11 @@ using ::std::tuple_size;
 # endif
 #elif __GNUC__ >= 4 || defined(__clang__)
 # define GTEST_API_ __attribute__((visibility ("default")))
```

Comment:
Or only do this if GTEST_LINKED_AS_SHARED_LIBRARY is defined, that's probably the best fix.